code/modules/webhooks/webhook_submap.dm

```diff
@@ -7,10 +7,10 @@
 	var/submap_name = data && data["name"]
 	if(submap_name)
 		submap_name = ", [submap_name],"
-	var/desc = "A submap[submap_name] is currently available."
+	var/desc = "A submap[submap_name] is currently available"
```

Comment:
I can more or less agree with removing the periods from titles, but I really don't think removing them from the contents is good. It's a full sentence, it should be grammatically correct.